index.html

```
@@ -1827,6 +1827,186 @@ <h2 id="itemModalTitle">Add Inventory Item</h2>
+                  <span class="image-side-label">Obverse</span>
+                  <div class="image-card">
+                    <div
+                      id="itemImagePreviewObv"
+                      class="image-upload-preview"
+                      style="display: none"
+                    >
+                      <img id="itemImagePreviewImgObv" alt="Obverse preview" />
+                    </div>
+                    <div class="image-card-actions">
+                      <input
+                        type="file"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)